swiftui/App/Views/OpenRealmView.swift

```diff
@@ -1,11 +1,27 @@
 import SwiftUI
 import RealmSwift
 
+// :state-uncomment-start: flexible-sync
+// enum SubscriptionState {
+//     case initial
+//     case completed
+// }
+// :state-uncomment-end:
+
 /// Called when login completes. Opens the realm asynchronously and navigates to the Todos screen.
 struct OpenRealmView: View {
+    // :state-uncomment-start: flexible-sync
+    // let user = realmApp.currentUser
+    // @State var subscriptionState: SubscriptionState = .initial
+    // :state-uncomment-end:
+    // :state-start: partition-based-sync
     // By leaving the `partitionValue` an empty string, we use the
     // `.environment(\.partitionValue, user.id)` passed in from the `ContentView`
     @AsyncOpen(appId: theAppConfig.appId, partitionValue: "", timeout: 4000) var asyncOpen
+    // :state-end:
+    // :state-uncomment-start: flexible-sync
+    // @AsyncOpen(appId: testAppId, timeout: 2000) var asyncOpen
```

Comment:
This probably still needs theAppConfig.appId

Reply:
Good catch - sorry about that!